### PR TITLE
Consent form updated to use jobTitle and companyName parameters

### PR DESCRIPTION
**Pull request by mobimation-gunnar**
AF Connect API updated to include Job Title and Company Name
needed for the consent form in approving use
of AF user profile data.
These values arrive part of a data sink.
```
      <div><b>JobTechDev</b> får bara använda informationen för följande syfte:</div>
      <ul style="margin-top: 16px">
          <% data.sink.purposeOfUse.forEach((purpose) => { %>
              <li><%= purpose %></li>
          <% }) %>
      </ul>
```

af-connect handles rendering of the consent form.

### CODE REVIEW

**Review by ixuz — CHANGES_REQUESTED** [review]
1. Render `<%= data.sink.sinkName %>` here: https://github.com/MagnumOpuses/af-connect/blob/forrr-new-params-consent/app/views/partials/consentForm.ejs#L174
2. Render `<%= data.sink.sinkName %>` here: https://github.com/MagnumOpuses/af-connect/blob/forrr-new-params-consent/app/views/partials/consentForm.ejs#L177

**Comment by mobimation-gunnar** [comment]
See those double pushes. I got confused. The first one 
b08ef38 is probably sufficient.

File: app/views/partials/consentForm.ejs [hunk 1]
```diff
@@ -141,30 +141,28 @@
...
+                      <div><b><%= data.sink.companyName %></b> får bara använda informationen för följande syfte:</div>
+                      <ul style="margin-top: 16px">
+                          <li>Ansökan till roll '<%= data.sink.jobTitle %>' på '<%= data.sink.companyName %>'</li>
+                      </ul>
+                  </div>
+              </div>
+
+              <div style="padding: 16px; width: 60%; margin: auto; text-align: left;">
+                  <div><input id="secrecyAgreement" type="checkbox" onclick="secrecyAgreement();"> Jag förstår att denna process häver sekretessen för de uppgifter jag väljer att dela.</div>
+                  <div><input id="transferAgreement" type="checkbox" onclick="transferAgreement();"> Jag ger mitt medgivande att Arbetsförmedlingen får hantera och överföra dessa uppgifter till <b><%= data.sink.companyName %></b>.</div>
```
Comment (ixuz) [inline app/views/partials/consentForm.ejs:153]:
```suggestion
                  <div><input id="transferAgreement" type="checkbox" onclick="transferAgreement();"> Jag ger mitt medgivande att Arbetsförmedlingen får hantera och överföra dessa uppgifter till <b><%= data.sink.sinkName %></b>.</div>

```

File: app/views/partials/consentForm.ejs [hunk 2]
```diff
@@ -141,30 +141,28 @@
...
+                  <div><input id="reviewAgreement" type="checkbox" onclick="reviewAgreement();"> Jag har granskat uppgifterna och bekräftar att inga oönskade uppgifter ingår i överföringen till <b><%= data.sink.companyName %></b>.</div>
```
Comment (ixuz) [inline app/views/partials/consentForm.ejs:154]:
```suggestion
                  <div><input id="reviewAgreement" type="checkbox" onclick="reviewAgreement();"> Jag har granskat uppgifterna och bekräftar att inga oönskade uppgifter ingår i överföringen till <b><%= data.sink.sinkName %></b>.</div>

```

**Review by ixuz — APPROVED** [review]
Looks good

**Pull request merged**